src/components/Forms/ContactForm/ContactForm.js

```diff
@@ -48,111 +48,170 @@ function ContactForm() {
       setLoading(false)
     }
   }
-
-  return (
-    <form id="s2do-form" onSubmit={handleSubmit(onSubmit)}>
-      <div className="col-md-4">
-        <div className="form-group">
-          <label htmlFor="InputName" className="dark-text">
-            Your Display Name
-            <sup>*</sup>
-          </label>
-          <input
-            type="text"
-            className="form-control"
-            name="name"
-            id="InputName"
-            placeholder="Jody"
-            ref={register({ required: true })}
-          />
+  if (props.emailMessage) {
```

Comment:
Can we do a different prop name here that describes we are using a smaller set of the inputs in the form?

Reply:
Sure can, any suggestions? I chose emailMessage as those are the 2 fields being used.

Reply:
Something describes the what the prop is used for. This is a boolean based prop so something like enable email and message Inputs. Or disable depending on how it is used.

Reply:
Something describes the what the prop is used for. This is a boolean based prop so something like enable email and message Inputs. Or disable depending on how it is used

Reply:
Something that describes what the prop is doing..`displayEmailAndMesssageInputs` or disable depending on how the prop is used.